src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+
+  &__img {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 40px;
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-left: 16px;
+    align-self: flex-start;
+    font-weight: 400;
+    font-size: 10px;
+    color: #616070;
+  }
+  
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    width: 166px;
```

Comment:
do not hardcode width for every element(DRY), its can be done done without it
or at least use var